geoserver/webapp/src/docker/docker-entrypoint.d/00-geoserver-bootstrap

```diff
@@ -16,4 +16,9 @@ else
 
     echo 'Change log config according to docker setup'
     sed -i 's:stdOutLogging>false:stdOutLogging>true:g' /mnt/geoserver_datadir/logging.xml
+
+    # geowebcache config need to be placed in geoserver_tiles volume.
+    # copying it directly in the Dockerfile doesn't work. Copying it at runtime.
```

Comment:
geowebcache config needs to be placed into the geoserver_tiles volume.
We cannot copy the file via the Dockerfile at build time, so we are doing it at runtime.